implementations/rust/ockam/ockam_command/src/run/parser.rs

```diff
@@ -174,8 +174,8 @@ pub struct Config {
 pub struct NodeConfig {
     #[serde(rename(deserialize = "depends-on"))]
     pub depends_on: Option<String>,
-    #[serde(rename(deserialize = "enroll-ticket"))]
-    pub enroll_ticket: Option<String>,
+    #[serde(rename(deserialize = "enrollment-ticket"))]
```

Comment:
Does this have ramifications for the BATS suite?

Reply:
I don't think so, but I'll double check it.

Reply:
If it looks clear to you, I'll hit resolve. CI should catch it if not 🤞🏻